src/libraries/Common/tests/System/Net/Prerequisites/NetCoreServer/NetCoreServer.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>$(AspNetCoreAppCurrent)</TargetFramework>
```

Comment:
Why `<TargetFramework>$(AspNetCoreAppCurrent)</TargetFramework>`  over `<TargetFramework>$(NetCoreAppCurrent)</TargetFramework>` or `<TargetFramework>$(NetCoreAppCurrent)</TargetFramework>`? 
Though they evaluate to the same value at the moment, it seems like `AspNetCoreAppCurrent` is only used by the wasm debugger, so it "feels" like one of the other properties should be used here but I'm not sure